network/mvpeernet.cpp

```diff
@@ -315,9 +318,15 @@ bool CMvPeerNet::HandlePeerHandshaked(CPeer *pPeer,uint32 nTimerId)
             setDNSeed.insert(ep);
         }
 
-        SetNodeData(ep,boost::any(pMvPeer->nService));
     }
 
+    /*Warning*/
+    if(!AddPeerMacAddress(pPeer, pMvPeer->remoteMacAddress, pMvPeer->IsInBound()))
```

Comment:
The problem is here: AddPeerMacAddress returned false will lead to reduce 10 points with RemovePeer(PROTOCOL_ERROR), It is Inconsistent with the original strategy（reduce 0 points）